mqtt-level-store.js

```diff
@@ -19,38 +19,112 @@ function Store (options) {
   }
 }
 
+const zeroPadding = (function () {
+  // width and padding calculates only once
+  const width = parseInt(Number.MAX_SAFE_INTEGER).toString(10).length
+  const padding = new Array(width + 1).join('0')
+  return function (num) {
+    return (padding + num).slice(-width)
+  }
+})()
+
 Store.prototype.put = function (packet, cb) {
-  this._level.put('' + packet.messageId, packet, this._levelOpts, cb)
+  var date = zeroPadding(Date.now())
```

Comment:
even though is slower, I would prefer to use `new Date().toISOString()`. That is lexicographically safe.

Reply:
Ok, I will replace.